go-fuzz-build/main.go

```diff
@@ -582,3 +618,28 @@ func main() {
 	dep.Main(target.%v)
 }
 `
+
+var mainSrc_libFuzzer = `
+package main
+
+import (
+	"C"
+	"unsafe"
+	target "%v"
+	dep "go-fuzz-dep"
+)
+
+
+//export gofuzz_Run
+func gofuzz_Run(input []byte) {
```

Comment:
The common convention for all Go code is that identifiers don't contain "_" and use Pascal/camelCase instead. Please rename this and gofuzz_Init.

Reply:
And mainSrc_libFuzzer